sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_auth_events_v1/query.sql

```diff
@@ -5,15 +5,16 @@ SELECT
         jsonPayload.* REPLACE (
           (
             SELECT AS STRUCT
-              jsonPayload.fields.* EXCEPT (device_id, user_id) REPLACE(
+              jsonPayload.fields.* EXCEPT (user_id, device_id, deviceid) REPLACE(
                 -- See https://bugzilla.mozilla.org/show_bug.cgi?id=1707571
                 CAST(NULL AS FLOAT64) AS emailverified,
                 CAST(NULL AS FLOAT64) AS isprimary,
                 CAST(NULL AS FLOAT64) AS isverified,
                 -- casting id as field type in source tables inconsistent
                 CAST(jsonPayload.fields.id AS STRING) AS id
               ),
-              TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id
+              TO_HEX(SHA256(jsonPayload.fields.user_id)) AS user_id,
+              COALESCE(jsonPayload.fields.device_id, jsonPayload.fields.deviceid) AS device_id
```

Comment:
why are we only fixing `device_id` and `deviceid` in nonprod?

Reply:
good point, I will make that tweak now.